CCR-1166 add new cube for covered countries

dbianco opened the pull request.

Commit:
CCR-1166 add new cube for covered countries

Commit:
CCR-1166 replace tabs with spaces

Commit:
CCR-1166 fix missing columns

nmandrescu committed:
Merge remote-tracking branch
'origin/feature/CCR-1206/optimize-schema-definitions' into
feature/CCR-1166/newCube

Conflicts:
	saiku-webapp/src/main/resources/ccrs-mondrian-cp1.xml

nmandrescu committed:
CCR-1207 CCR-1166 use simplified schema definition

nmandrescu committed:
Merge remote-tracking branch 'origin/develop' into feature/CCR-1166/newCube

nmandrescu committed:
Merge remote-tracking branch 'origin/feature/CCR-1206/optimize-schema-definitions' into feature/CCR-1166/newCube

nmandrescu committed:
Merge remote-tracking branch
'origin/bug/CCR-1237/shared-YesNoTable-used-multiple-times' into
feature/CCR-1166/newCube

nmandrescu committed:
CCR-1237 CCR-1166 fix for same category usage within the cube

developster reviewed (changes requested):
Dimensions for 1.B.1 (Covered Country) and 1.B.1.1 (Covered Country Income Status) are missing.

Commit:
Merge branch 'develop' into feature/CCR-1166/newCube

# Conflicts:
#	saiku-webapp/src/main/resources/ccrs-mondrian-cp1.xml

Commit:
CCR-1166
Merge branch 'feature/CCR-1166/newCube' of https://github.com/devgateway/ccrs-saiku into feature/CCR-1166/newCube

# Conflicts:
#	saiku-webapp/src/main/resources/ccrs-mondrian-cp1.xml

Add fixes from Jira bugs
BUG1
Number of Covered Countries is a default measure. It counts the number of countries from Countries section of the form.
Count of 'Is it a DAO Country' is the measure that was explicitly requested. It counts number of countries that are DAO.

BUG2
Country dimension stands for country that submitted the CP.
Indeed, dimensions 1.B.1 (Covered Country) and 1.B.1.1 (Covered Country Income Status) are missing.

Commit:
Merge branch 'develop' into feature/CCR-1166/newCube

Commit:
CCR-1262 Saiku CP1 Covered Countries: IndexOutOfBoundsException
CCR-1261 Saiku CP1 Covered Countries: repeated Variable name in the table

Commit:
CCR-1166  fix for "Year of becoming a DAO country"

Commit:
CCR-1166 remove "uncategorized" filter from covered country query
